auslib/db.py

```diff
@@ -1923,6 +1923,15 @@ def getBlob():
 
         return blob
 
+    def getReleaseBlobs(self, names, transaction=None):
+        dataVersions = self.select(where=[self.name.in_(names)], columns=[self.name, self.data_version], transaction=transaction)
+        blobs = {version['name']: version for version in dataVersions}
+
+        q = self.select(where=[self.name.in_(names)], columns=[self.data], transaction=transaction)
```

Comment:
I _think_ it would be better to use `getReleaseBlob` (https://github.com/mozilla/balrog/blob/99e230f9eef15c29c8cdb9000978eeec46c63a9e/auslib/db.py#L1879) for this, because it will automatically use the in-memory cache.

Reply:
Hi @mozbhearsum I've tried using `getReleaseBlob` but this significantly increases response time to `>12000ms`. That's if I loop through a set of both `mappings` and `fallbackMappings` then call `getReleaseBlob` on each. I've also tried getting all blobs in one query and this takes about `~500ms` which is pretty reasonable but this means I would have to modify the `getReleaseBlob` function return type and also omit caching. What do you think about this?

Reply:
Does the response time go up on just the first request, or all subsequent requests, too? Either way, I'm open to grabbing them all in one query. I'd like to chat about the details of that first, though.

Nice investigative work!

Reply:
The first request is actually `~3600ms` the rest are `~2000ms`, when testing on local machine.

Reply:
That's still pretty slow, and would be slower in production :(. Sounds like we should pursue your idea for improvement.

Reply:
Alright, I'm going to try doing that. Thank you!